src/asset-manager.js

```diff
@@ -59,8 +59,11 @@ function main(container, _runner) {
 		client.reevaluateContext();
 	};
 
-	client.getActiveKeys('web').listen(function (keys) {
+	let hasRunRedirect = false
+
+	client.getActiveKeys('').listen(function (keys) {
```

Comment:
Here I removed 'web', because in fact, we want to have all keys. As we discussed before, for redirect variants we want to handle variant that are located not only under `web` group. So, for this, I make a call to get all keys. 
Then for js variants we are filtering them out (we want to handle only 'web' ones). Below on the L90, I am passing to use all the keys for handling redirect variants.

Here is an example:
-web
 -varGroup1
  -javascript variant

-notweb
 -varGroup2
  - redirect variant

In this case both variants (javascript and redirect) will be handled.

@robertsevernsentient Am I missing something?

Reply:
Discussed offline - we only want to handle `web` variants